src/coreclr/System.Private.CoreLib/src/System/Enum.CoreCLR.cs

```diff
@@ -13,8 +12,32 @@ public abstract partial class Enum
         [LibraryImport(RuntimeHelpers.QCall, EntryPoint = "Enum_GetValuesAndNames")]
         private static partial void GetEnumValuesAndNames(QCallTypeHandle enumType, ObjectHandleOnStack values, ObjectHandleOnStack names, Interop.BOOL getNames);
 
-        [MethodImpl(MethodImplOptions.InternalCall)]
-        private static extern object InternalBoxEnum(RuntimeType enumType, long value);
+        private static unsafe object InternalBoxEnum(RuntimeType enumType, long value)
+        {
+            MethodTable* pMethodTable = (MethodTable*)enumType.m_handle;
+            object obj = RuntimeHelpers.AllocateUninitializedObject(pMethodTable);
+            ref byte dataRef = ref obj.GetRawData();
+
+            switch (pMethodTable->GetNumInstanceFieldBytes())
+            {
+                case 1:
+                    dataRef = (byte)value;
+                    break;
+                case 2:
+                    Unsafe.As<byte, ushort>(ref dataRef) = (ushort)value;
+                    break;
+                case 3:
```

Comment:
3?

Reply:
Whoops, small typo 😅
Fixed in https://github.com/dotnet/runtime/pull/70181/commits/483c9df45de4d3291f6322c66e5657c63341a648.